pages/common/gh-repo.md

```diff
@@ -0,0 +1,20 @@
+# gh repo
+
+> Work with GitHub repositories on the command line.
+> More information: <https://cli.github.com/manual/gh_repo>.
+
+- Create a new repository (if the repository name is not set, it will default to the name of the current directory):
+
+`gh repo create {{name}}`
+
+- Clone a repository:
+
+`gh repo clone {{repository}}`
```

Comment:
You should include the `owner` if the repo is not in your account:
```suggestion
`gh repo clone {{owner}}/{{repository}}`
```
Maybe you can add an extra example for clone the repositories in the logged in account.